src/main/java/com/lothrazar/cyclic/block/generatorpeat/TilePeatGenerator.java

```diff
@@ -88,14 +88,13 @@ public void tick() {
       return;
     }
     if (this.isFull()) {
+      this.tickCableFlow();
       return;
     }
-    if (energy.getEnergyStored() + 1 >= energy.getMaxEnergyStored()) {
```

Comment:
Note that I removed the `energy.getEnergyStored() + 1 >= energy.getMaxEnergyStored()` statement that followed a check on `this.isFull`, as it seemed to be a duplicate that did not affect functionality when testing.